sdk/storage/storage-queue/src/utils/utils.common.ts

```diff
@@ -114,6 +114,28 @@ export function getURLQueries(url: string): { [key: string]: string } {
   return queries;
 }
 
+function getDevConnString(connectionString: string): string {
+  // Development Connection String
+  // https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string#connect-to-the-emulator-account-using-the-well-known-account-name-and-key
+  const partialConnectionString = `DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;
+        AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;`;
+
+  if (connectionString.search("DevelopmentStorageProxyUri=") !== -1) {
+    // CONNECTION_STRING=UseDevelopmentStorage=true;DevelopmentStorageProxyUri=http://myProxyUri
+    let proxyUri = "";
+    const matchCredentials = connectionString.split(";");
+    for (const element of matchCredentials) {
+      if (element.trim().startsWith("DevelopmentStorageProxyUri=")) {
+        proxyUri = element.trim().match("DevelopmentStorageProxyUri=(.*)")![1];
+      }
+    }
+    return partialConnectionString + `QueueEndpoint=${proxyUri}/devstoreaccount1;`;
```

Comment:
Is this going to work? I was thinking of `proxyUri` is meant to be passed to the pipeline's proxy settings

Reply:
Oh, could be. I was assuming that it's of the emulator.

Reply:
Updated!